setup.py

```diff
@@ -60,11 +60,12 @@ def get_library_dirs():
                 "src/triangulations.cpp",
                 "src/skeletonization.cpp",
                 "src/reconstruction.cpp",
+                "src/polygonal_surface_reconstruction.cpp",
             ]
         ),
         include_dirs=["./include", get_eigen_include(), get_pybind_include()],
         library_dirs=[get_library_dirs()],
-        libraries=["mpfr", "gmp"],
+        libraries=["mpfr", "gmp", "libscip"],
```

Comment:
`libscip` => `scip`

Reply:
On Windows this results in:
`LINK : fatal error LNK1181: cannot open input file 'scip.lib'`

Even though it is installed through conda:
<img width="494" alt="image" src="https://github.com/compas-dev/compas_cgal/assets/18013985/5e0ad095-1d58-4cb4-bf08-27e22c98fd58">


Can I specify like this:

```python
def get_scip_library():
    if windows:
        return "libscip"
    return "scip"

ext_modules = [

    Extension(
        "compas_cgal._cgal",
        sorted(
            [
                "src/compas_cgal.cpp",
                "src/compas.cpp",
                "src/meshing.cpp",
                "src/booleans.cpp",
                "src/slicer.cpp",
                "src/intersections.cpp",
                "src/measure.cpp",
                "src/subdivision.cpp",
                "src/triangulations.cpp",
                "src/skeletonization.cpp",
                "src/reconstruction.cpp",
                "src/polygonal_surface_reconstruction.cpp",
            ]
        ),
        include_dirs=["./include", get_eigen_include(), get_pybind_include()],
        library_dirs=[get_library_dirs()],
        libraries=["mpfr", "gmp", get_scip_library()],
        language="c++",
    ),
]


```